webkitgtk-2.53.4/Source/WebKit/WebProcess/WebPage/CoordinatedGraphics/NonCompositedFrameRenderer.cpp:337:93: error: reference to 'Region' is ambiguous
  337 | void NonCompositedFrameRenderer::foreachRegionInDamageHistoryForTesting(Function<void(const Region&)>&& callback) const
      |                                                                                             ^~~~~~
In file included from /export/home/gweiss/git/oi-userland/component

build/amd64/WebCore/PrivateHeaders/WebCore/Region.h:38:7: note: candidates are: 'class WebCore::Region'
   38 | class Region {
      |       ^~~~~~

--- webkitgtk-2.53.4/Source/WTF/wtf/PlatformEnableGlib.h.old	2026-07-26 23:01:32.879592988 -0400
+++ webkitgtk-2.53.4/Source/WTF/wtf/PlatformEnableGlib.h	2026-07-26 23:01:54.625602709 -0400
@@ -72,5 +72,5 @@
 #endif
 
 #if ENABLE(WPE_PLATFORM) || PLATFORM(GTK)
-#define ENABLE_DAMAGE_TRACKING 1
+#define ENABLE_DAMAGE_TRACKING 0
 #endif
